pom.xml

```diff
@@ -219,8 +219,9 @@
 
     <modules>
         <module>archetype</module>
-        <module>cdi2</module>
         <module>core</module>
+        <module>cdi2</module>
+        <module>jakarta-cdi</module>
```

Comment:
Please keep these in alphabetical order

Reply:
Ok but it does not help reactor work :(

Reply:
Maven is smart enough to figure out the reactor order. I'd like them in alphabetical order so I can quickly read it.